mlir/python/mlir/dialects/transform/structured.py

```diff
@@ -140,6 +140,57 @@ def __init__(
         )
 
 
+@_ods_cext.register_operation(_Dialect, replace=True)
+class FuseOp(FuseOp):
+    """Specialization for FuseOp class."""
+
+    @overload
+    def __init__(
+        self,
+        target: Union[Operation, Value, OpView],
+        *,
+        sizes: Optional[Union[DynamicIndexList, ArrayAttr]] = None,
+        interchange: OptionalIntList = None,
+        loc=None,
+        ip=None,
+    ):
+        ...
```

Comment:
I think you need to explicitly list both overloads. And use exactly the same names for arguments, otherwise the linter complains.